tests/test_doc_maker.py

```diff
@@ -38,226 +14,137 @@ class TestCreateClass(unittest.TestCase):
     """
 
     def setUp(self):
-        self.doc = hydra_doc_sample.doc
-
-    @patch('hydra_python_core.doc_maker.re')
-    def test_validations(self, mock_re):
```

Comment:
Rather than deleting test, if you have added other things in the doc_maker to handle such cases, or if there are errors raised then catch those. It's a good idea to test negative cases as well. I see a lot of the tests removed, I wonder if all those use cases are not relevant anymore.

Reply:
Most of the tests that were removed were no longer valid. The new test file contains tests for every function like it was earlier. Since most of the functions were removed from the module, hence I had to remove the tests as well.